lighthouse-core/gather/gather-runner.js

```diff
@@ -106,7 +106,9 @@ class GatherRunner {
     log.log('status', 'Initializing…');
     const resetStorage = !options.settings.disableStorageReset;
     // Enable emulation based on settings
-    await driver.assertNoSameOriginServiceWorkerClients(options.requestedUrl);
+    if (!options.settings.skipNavigation) {
+      await driver.assertNoSameOriginServiceWorkerClients(options.requestedUrl);
+    }
     const userAgent = await driver.getUserAgent();
     gathererResults.UserAgent = [userAgent];
     GatherRunner.warnOnHeadless(userAgent, gathererResults);
```

Comment:
there's other setup stuff that we can probably skip in this block, but if it's not necessary for you I guess we can pick that up later 👍